### DISCUSSION TITLE
Importing sub-config files

### DISCUSSION THREAD

**Discussion by TheZoq2** [hyprwm/Hyprland#3651] [Q&A] [answered]
Apologies if I've missed some documentation, but I can't figure out if there is a way to import files in the hyprland config.

My use case for this is that I use the same config across 3 different computers with varying monitor setups. Having the monitor setup be part of the main config is a huge pain whenever I want to make changes, so being able to `include monitors.conf`, with mointor.conf not being committed to my dotfiles repo would be really good. Is there a way to do this?

**Comment by memchr** [comment, marked as answer]
https://wiki.hyprland.org/Configuring/Keywords/#sourcing-multi-file

globbing supported

**Reply by TheZoq2** [reply]
Thank you! I searched for `include` and `import` but didn't think of `source` :pensive:
